src/devtools/devtools.helper.js

```diff
@@ -0,0 +1,22 @@
+export function removeCyclicReferences(apps) {
+  // we must get rid of cyclic references before we can return apps -- otherwise, devtools gets an error because JSON.stringify doesn't resolve cyclic references
```

Comment:
Does devTools need to stringify the apps? React Components are also circular, what are they doing differently (or are they doing the same thing)

I'm not sure I fully understand why this is necessary